content/webapp/services/prismic/transformers/articles.ts

```diff
@@ -130,10 +130,19 @@ export function transformArticle(document: ArticlePrismicDocument): Article {
   ].filter(isNotUndefined);
 
   const contributors = transformContributors(document);
+  // This is getting a little unwieldy, as i am learning there
+  // are a few formats we consider to be 'articles' even if their Prismic 'format'
+  // doesn't return them as 'Article' e.g. Book extracts
+  // TODO: get definitive list of what we consider to be article and refactor the below function to account for that
   const isArticleOrSerialFormat = format
```

Comment:
Maybe it's time to rename this variable to `showReadingTime`, so it doesn't drift from what it does?

Reply:
agreed! Thank you